File: Hrm.Domain/PostingOrderInfo.cs
   using Hrm.Domain.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hrm.Domain
{
    public class PostingOrderInfo : BaseDomainEntity
    {

        public int PostingOrderInfoId { get; set; }
       
        public int? EmpId { get; set; }
        public int? DepartmentId { get; set; }
        public int? DesignationId { get; set; }
        public int? SubBranchId { get; set; }
        public int? SubDepartmentId { get; set; }
        public int? OfficeId { get; set; }
        public string? OfficeOrderNo { get; set; }
        public DateOnly? OfficeOrderDate { get; set; }
        public string? OrderOfficeBy { get; set; }
        public string? TransferSection { get; set; }
        public string? ReleaseType { get; set; }
        public int? MenuPosition { get; set; }
        public bool IsActive { get; set; }
    }
}

File: ClientApp/src/app/views/basic-setup/model/posting-order-info.ts
export class PostingOrderInfo {
    postingOrderInfoId: number = 0;
    empId: number = 0;
    departmentId:any=null;
    subBranchId:number=0;
    subDepartmentId:number=0;
    designationId:any=null;
    designationName:string="";
    officeName:string="";
    departmentName:string="";
    officeId: any=null;
    officeOrderNo:string="";
    officeOrderDate:Date= new Date();
    orderOfficeBy:string="";
    transferSection:string="";
    releaseType:string="";
    menuPosition: number = 0;
    isActive: boolean = true
}

